src/main/java/uk/gov/digital/ho/hocs/casework/domain/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -75,7 +75,7 @@ public ResponseEntity handle(HttpMessageNotReadableException e) {
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity handle(Exception e) {
-        log.error("Exception: {}", e.getMessage(), value(EVENT, UNCAUGHT_EXCEPTION));
+        log.error("An error occurred", e);
```

Comment:
We still need `value(EVENT, UNCAUGHT_EXCEPTION)` as that seems to be the way be define logs using this key value pair.

Reply:
I have readded this.